server/models/Experiment.ts

```diff
@@ -0,0 +1,38 @@
+import { DataTypes, Model } from "sequelize"
+import sequelize from "../utils/sequelize"
+
+const experimentStatusEnum = DataTypes.ENUM("Draft", "Submitted", "Accepted")
+class Experiment extends Model {}
+
+Experiment.init(
+  {
+    // Model attributes are defined here
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    experimentStatus: {
+      type: experimentStatusEnum,
+      allowNull: false,
+      defaultValue: "Draft",
+    },
+    // Duration in minutes
+    duration: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      defaultValue: 0,
```

Comment:
I think a default value of 0 could lead to unexpected outcomes when you accidentally don't set the duration when creating a new experiment. In my opinion it would be better to leave the default value for the duration away.